www/src/components/publisher/CreatePublisherModal.tsx

```diff
@@ -257,9 +257,11 @@ function CreatePublisherModal({ open, onClose }: CreatePublisherModalProps) {
         )}
       </Flex>
       {!!preMutationError && (
-        <Alert severity="error">
-          {preMutationError}
-        </Alert>
+        <Alert
```

Comment:
Isn't this reintroducing the alert instead of using the design system's?

Reply:
We deprecated the DS Alert component

Reply:
oh gotcha